appveyor.yml

```diff
@@ -23,7 +23,7 @@ matrix:
   fast_finish: true
 
 install:
-  - ps: Install-Product node $env:nodejs_version x64
+  - ps: Update-NodeJsInstallation $env:nodejs_version $env:TARGET_ARCH
```

Comment:
Hmmm? How come trying to install the x64 version of node on x86 Windows wasn't previously causing problems?